wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -0,0 +1,341 @@
+- `osu!taiko`
+- `osu!catch`
+- `osu!mania`
+
+*Notice: `osu!standard` is used to maintain consistency when referring to the game mode. Folder names, however, must use `osu!` even if it is referring to the game mode.*
+
+Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing about said game mode's previous name.
+
+---
+
+Articles such as the [Ranking Criteria](/wiki/Ranking_Criteria) may use `osu!` instead of `osu!standard`.
+
+## osu!
+
+The name of the game, osu!, must not be capitalised or italicised. The osu! official branding must not use any spaces. Examples include:
+
+- `osu!academy`
+- `osu!api`
+- `osu!catch`
+- `osu!direct`
+- `osu!keyboard`
+- `osu!mania`
+- `osu!store`
+- `osu!stream`
+- `osu!supporter`
+- `osu!tablet`
+- `osu!taiko`
+- `osu!talk`
+- `osu!tourney`
+
+Other brandings that are not covered by the osu! official branding must use a space. Examples include:
+
+- `osu! tournaments`
+- `osu! community`
+- `osu! chat`
+- `osu! client`
+- `osu! wiki`
+
+User titles that include `osu!` as part of the title name must be capitalised. Examples include:
+
+- `osu! Alumni`
+- `osu! Champion`
+
+---
+
+`osu!` should not end the sentence. If it does however, use a full stop (`.`) (or question mark (`?`), if appropriate) to get `osu!.` (or `osu!?`).
+
+`osu!` must not be followed by an exclamation mark (`!`) (i.e. `osu!!`). It must be rewritten to ensure that this will not occur.
+
+## Terminology
+
+These words must be spelt as follows (spacing must match):
+
+- `approach circle`
+- `game mode` (or `mode`)
+- `game mod` (or `mod`)
+- `gameplay`
+- `hit burst`
+- `hit circle`
+- `hitsound`
+- `in-game`
+- `playstyle`
+- `slider tick`
+- `slider ball`
+- `slider path`
+
+---
+
+Some words have variants. Their preferred spelling must be used and is as follows:
+
+- `beatmap` instead of `map`.
+- `creator` instead of `beatmapper` or `mapper`.
+- `mapped` instead of `beatmapped`.
+- `BN` or `Beatmap Nominators` when referring to the *Beatmap Nominators*.
+- `sign in` instead of `log in`, unless the name of a button or link says otherwise.
+- `sign out` instead of `log out`, unless the name of a button or link says otherwise.
+- `register` instead of `sign up`, unless the name of a button or link says otherwise.
+
+### Tournament articles
+
+These words must be spelt as follows (spacing must match):
+
+- `NoMods`, `NoMod`, or `NM`
+- `FreeMods`, `FreeMod`, or `FM`
+- `Tiebreaker`
+
+## Abbreviations, acronyms, and initialisms
+
+Abbreviations, acronyms, and initialisms must have their meaning written out upon their first occurrence. Other occurrences are optional, but done only when necessary. The following is an example:
+
+```markdown
+The NC (Nightcore) mod is similar to the DT (Double Time) mod because both NC and DT increase the speed of the music by 50%. However, NC will change the pitch of the music and adds a clap and finish to the beat.
```

Comment:
```suggestion
The NC (Nightcore) mod is similar to the DT (Double Time) mod because both NC and DT increase the speed of the music by 50%. However, NC will change the pitch of the music and add a clap and finish to the beat.
```